api/Navigator.json

```
@@ -433,7 +433,9 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/canShare",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "89",
+              "partial_implementation": true,
+              "notes": "Not supported on macOS, see <a href='https://bugs.chromium.org/p/chromium/issues/detail?id=1144920'>bug 1144920</a>."
```

```suggestion
              "notes": "Not supported on macOS, see <a href='https://crbug.com/1144920'>bug 1144920</a>."
```

The lint is failing because of this.